### ISSUE TITLE
Clean instal of wf-artic 0.3.9

### ISSUE THREAD

**Issue by adamkoletis** [epi2me-labs/wf-artic#39]
Hi Matt,
Please I need to have a clean install of 0.3.9. So when I run nextflow info epi2me-labs/wf-artic to have only version until 0.3.9 and not 0.3.10.
I know that i can get the latest version and with the appropriate flag to run the previous version. But is something that I do not want at this moment.
I have tried to delete wf-artic v0.3.10 for /usr/local/bin/ folder but when I am downloading the zip file wf-artic0.3.9 and I am transferring the wf-artic folder into /usr/local/bin/ then I run again the info and I am still getting the latest version.
I appreciate any help

**Comment by mattdmem** [comment]
Morning!

Does this work? `nextflow run epi2me-labs/wf-artic -r v0.3.9`

If I do this I get:

```
nextflow run epi2me-labs/wf-artic -r v0.3.9
N E X T F L O W  ~  version 21.10.4
Launching `epi2me-labs/wf-artic` [disturbed_leakey] - revision: 8d13aab29f [v0.3.9]
```

Thanks

Matt

**Comment by adamkoletis** [comment]
Hi,
I have tried how you have suggest me but after the info I am getting again v0.3.10.
My point is that I do not want to have v.03.10.
I have installed previously the latest version. After manually I deleted from /usr/local/bin folder and then I tried to download your previous version and to install in the same folder....the issue is that what ever I am trying I am still at the info getting v.0.3.10 Is possible that deleting the folder that contains wf-artic 0.3.10 it was not enough?

**Comment by mattdmem** [comment]
I understand that you want to run `v0.3.9` and the command I posted should do that. Where is it reporting that it is running `v0.3.10`?

Nextflow caches workflows in `~/.nextflow/assets/epi2me-labs/wf-artic` which you could remove and try the command above again.

We also have some more instructions here: https://labs.epi2me.io/managing-epi2me-labs-workflows-with-the-nextflow-command-line-interface/

**Comment by adamkoletis** [comment]
![Screenshot from 2022-02-07 13-57-29](https://user-images.githubusercontent.com/93767749/152792681-bccbf2ce-6d8d-4fb2-b5cc-50464fd0a7c8.png)

**Comment by adamkoletis** [comment]
Initially I have deleted ~/.nextflow/assets/epi2me-labs/wf-artic
Then I have followed the commands and then when I have runned the info I had again v 0.3.10

**Comment by mattdmem** [comment]
Yes, it shows `v0.3.10` is available, but when you ran with `-r v0.3.9` it is running with `v0.3.9` - can you please run some data using that command and check the workflow report for the version?

**Comment by adamkoletis** [comment]
Yes I will do right now and I will update you thank you.

**Comment by adamkoletis** [comment]
Hi, still I have some issues. I will send you screenshots in some minutes.
Thanks

**Comment by adamkoletis** [comment]
Yesterday I have followed exactly your command lines and I was able to get results. 
Used -r v0.3.9 and under the flag samples I have added the sample sheets
Then with out to add -r I have run all the command line and I show the change of version to the newer one. Ofc I have added sample_sheet instead of samples.
Today that I have tried again the same commands of yesterday and switch between the 2 versions I am getting the follow error in screenshot
![Screenshot from 2022-02-08 10-38-14](https://user-images.githubusercontent.com/93767749/152964357-84c58143-0c98-442d-986d-b166a4f9a0aa.png)
Also with the new version we are receiving in the html file and the unclassified results
![Screenshot from 2022-02-08 10-08-03](https://user-images.githubusercontent.com/93767749/152964682-eece01e5-7e41-4133-b557-5be86bfc8455.png)

**Comment by mattdmem** [comment]
Looks like something to do with the fastq input and the samplesheet - can you check both to make sure:

1. The structure under `../basecalling/pass` is  `barcodeXX` etc
2. The samplesheet barcode matches the barcodes present in the fastq dir
3. If you move/delete the `unclassified` folder does that work?

**Comment by adamkoletis** [comment]
Ok I will try and I will update you again

**Comment by mattdmem** [comment]
I'm going to close this now as your original issue is solved, if you have any further problems please don't hesitate to raise a new issue.

**Comment by LianaKafetzopoulou** [comment]
Hi Matt,

Following up from Adam. To answer your points above:
- We have checked our folder structure and looks good - as you mention  under ../basecalling/pass is barcodeXX etc and also an unclassified folder 
- Our sample_sheet  - no unclassified included in there 
- If we remove the unclassified folder then this results in a report without the unclassified included

This is something that we did not have until now - so all previous versions of wf-artic were generating reports without the unclassified folder despite the folder being present. It only seems to be showing up in the latests updates - is this on purpose?

Thanks a lot for your help!

Liana

Liana
